graph-service/analyze.py

```
@@ -0,0 +1,115 @@
+                projection_cypher["relationshipQuery"],
+            )
+            # project_result is a GraphCreateResult (mapping-like)
+            node_ct = project_result.get("nodeCount") if isinstance(project_result, dict) else getattr(project_result, "nodeCount", "?")
+            rel_ct = project_result.get("relationshipCount") if isinstance(project_result, dict) else getattr(project_result, "relationshipCount", "?")
+            logger.info(f"Projected graph with {node_ct} nodes, {rel_ct} relationships")
+
+            # Retrieve a Graph object handle for algorithm calls
+            graph = gds.graph.get(GDS_GRAPH_NAME)
+
+            # Community detection (Louvain)
+            communities = gds.louvain.mutate(graph, relationshipWeightProperty="weight", mutateProperty="community")
```

Ruff (0.11.9)

Local variable `communities` is assigned to but never used

Remove assignment to unused variable `communities`

(F841)